misc/examples.md

```diff
@@ -0,0 +1,11 @@
+# More examples
+
+<a href="https://github.com/taichi-dev/taichi/blob/master/examples/mpm_lagrangian_forces.py"><img src="https://github.com/taichi-dev/public_files/blob/6bd234694270c83baf97ba32e0c6278b8cf37e6e/taichi/lagrangian.gif" height="192px"></a>
+<a href="https://github.com/taichi-dev/taichi/blob/master/examples/taichi_sparse.py"><img src="https://github.com/taichi-dev/public_files/blob/6bd234694270c83baf97ba32e0c6278b8cf37e6e/taichi/sparse_grids.gif" height="192px"></a>
+<a href="https://github.com/taichi-dev/taichi/blob/master/examples/pbf2d.py"><img src="https://github.com/taichi-dev/public_files/blob/6bd234694270c83baf97ba32e0c6278b8cf37e6e/taichi/pbf.gif" height="192px"></a>
+<a href="https://github.com/taichi-dev/taichi/blob/master/examples/game_of_life.py"><img src="https://github.com/taichi-dev/public_files/blob/6bd234694270c83baf97ba32e0c6278b8cf37e6e/taichi/game_of_life.gif" height="192px"></a>
+
+<a href="https://github.com/taichi-dev/taichi/blob/master/examples/fem128.py"><img src="https://user-images.githubusercontent.com/20640597/87796122-8704e900-c87b-11ea-92ba-ff318e44736d.gif" height="192px"></a>
+<a href="https://github.com/taichi-dev/taichi/blob/master/examples/mass_spring_3d.py"><img src="https://user-images.githubusercontent.com/20640597/86505395-93f2f880-bdf6-11ea-955b-b9120ba8f10c.gif" height="192px"></a>
+<a href="https://github.com/taichi-dev/taichi/blob/master/examples/mass_spring_3d.py"><img src="https://user-images.githubusercontent.com/20640597/86887569-55797880-c12b-11ea-8b73-22b659864032.gif" height="192px"></a>
+<a href="https://github.com/taichi-dev/taichi/blob/master/examples/mpm3d.py"><img src="yuanming-could-you-do-me-a-favor-by-uploading-these-images?" height="192px"></a>
```

Comment:
@archibate Could you please upload the gif files to https://github.com/taichi-dev/public_files? You have write access to that repo :-)